### ISSUE TITLE
Null Object Reference during no-input moment at 16X

### ISSUE THREAD

**Issue by IBwWG** [larsiusprime/tdrpg-bugs#701]
A few moments after the screenshot at #699:

![dq-crash](https://cloud.githubusercontent.com/assets/16719964/14822959/f665748e-0bd0-11e6-946b-a5e70e420428.png)

```
FlxUIButton name = movies_btn
FlxUIButton name = music_btn
FlxUIButton name = sfx_btn
FlxUIButton name = party_btn
FlxUIButton name = btn_journal
FlxUIButton name = btn_town
FlxUIButton name = tips_btn
FlxUIButton name = btn_battle
FlxUIButton name = exit_btn
FlxUIButton name = options_btn
FlxUIButton name = menu_btn
FlxUIButton name = bonus_btn
Called from ApplicationMain::main ApplicationMain.hx line 159
Called from ApplicationMain::create ApplicationMain.hx line 71
Called from lime.app.Application::exec lime/app/Application.hx line 222
Called from lime._backend.native.NativeApplication::exec lime/_backend/native/Na
tiveApplication.hx line 136
Called from lime._backend.native.NativeApplication::handleRenderEvent lime/_back
end/native/NativeApplication.hx line 368
Called from lime.app.Event_Void_Void::dispatch lime/app/Event.hx line 145
Called from *::_Function_2_1 lime/app/Application.hx line 110
Called from lime.app.Application::render lime/app/Application.hx line 762
Called from openfl.display.Stage::render openfl/display/Stage.hx line 677
Called from openfl.display.DisplayObjectContainer::__broadcast openfl/display/Di
splayObjectContainer.hx line 355
Called from openfl.display.DisplayObject::__broadcast openfl/display/DisplayObje
ct.hx line 251
Called from openfl.events.EventDispatcher::__dispatchEvent openfl/events/EventDi
spatcher.hx line 237
Called from flixel.FlxGame::onEnterFrame flixel/FlxGame.hx line 529
Called from flixel.FlxGame::step flixel/FlxGame.hx line 662
Called from com.leveluplabs.tdrpg.Main::update com/leveluplabs/tdrpg/Main.hx lin
e 1462
Called from flixel.FlxGame::update flixel/FlxGame.hx line 751
Called from flixel.FlxState::tryUpdate flixel/FlxState.hx line 168
Called from com.leveluplabs.tdrpg.State_Battle::update com/leveluplabs/tdrpg/Sta
te_Battle.hx line 867
Called from com.leveluplabs.tdrpg.State_Battle::updateBasic com/leveluplabs/tdrp
g/State_Battle.hx line 914
Called from com.leveluplabs.tdrpg.BattleField::update com/leveluplabs/tdrpg/Batt
leField.hx line 876
Called from flixel.group.FlxTypedGroup::update flixel/group/FlxGroup.hx line 138

Called from flixel.group.FlxTypedGroup::update flixel/group/FlxGroup.hx line 138

Called from com.leveluplabs.tdrpg.CreatureSprite::update com/leveluplabs/tdrpg/C
reatureSprite.hx line 1490
Called from com.leveluplabs.tdrpg.TDRPGEntitySprite::update com/leveluplabs/tdrp
g/TDRPGEntitySprite.hx line 88
Called from flixel.FlxSprite::update flixel/FlxSprite.hx line 586
Called from com.leveluplabs.tdrpg.CreatureSprite::updateAnimation com/leveluplab
s/tdrpg/CreatureSprite.hx line 489
Called from com.leveluplabs.tdrpg.TDRPGEntitySprite::updateAnimation com/levelup
labs/tdrpg/TDRPGEntitySprite.hx line 76
Called from flixel.editors.EntitySprite::updateAnimation flixel/editors/EntitySp
rite.hx line 730
Called from flixel.FlxSprite::updateAnimation flixel/FlxSprite.hx line 594
Called from flixel.animation.FlxAnimationController::update flixel/animation/Flx
AnimationController.hx line 88
Called from flixel.animation.FlxAnimation::update flixel/animation/FlxAnimation.
hx line 224
Called from flixel.animation.FlxAnimation::set_curFrame flixel/animation/FlxAnim
ation.hx line 268
Called from flixel.animation.FlxBaseAnimation::set_curIndex flixel/animation/Flx
BaseAnimation.hx line 33
Called from flixel.animation.FlxAnimationController::set_frameIndex flixel/anima
tion/FlxAnimationController.hx line 733
Called from flixel.editors.EntitySprite::animationCallback flixel/editors/Entity
Sprite.hx line 869
Called from com.leveluplabs.tdrpg.DefenderSprite::onSweetFrame com/leveluplabs/t
drpg/DefenderSprite.hx line 634
Called from com.leveluplabs.tdrpg.DefenderSprite::launchAttack com/leveluplabs/t
drpg/DefenderSprite.hx line 2491
Called from com.leveluplabs.tdrpg.DefenderSprite::checkTargetDead com/leveluplab
s/tdrpg/DefenderSprite.hx line 1430
Called from com.leveluplabs.tdrpg.DefenderSprite::fixInvalidTarget com/levelupla
bs/tdrpg/DefenderSprite.hx line 2062
Called from com.leveluplabs.tdrpg.DefenderSprite::onOverlap com/leveluplabs/tdrp
g/DefenderSprite.hx line 596
Called from com.leveluplabs.tdrpg.DefenderSprite::considerAttack com/leveluplabs
/tdrpg/DefenderSprite.hx line 204
Called from com.leveluplabs.tdrpg.DefenderSprite::considerDualRangeAttack com/le
veluplabs/tdrpg/DefenderSprite.hx line 148
```

**Comment by IBwWG** [comment]
Not sure if it's related to #665.  It *was* pretty close to the end of the battle.

**Comment by larsiusprime** [comment]
Added a null check. Presumed fixed in 0.5.4.
